### PR TITLE
Add info about reports into HACKING.md

### PR DESCRIPTION
**Pull request by gentlecat**
:page_facing_up:

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -171,6 +171,26 @@ Now you should be able to use it to run the tests:
     $ prove -l t/qunit.t
 
 
+Reports
+-------
+
+[Reports](https://beta.musicbrainz.org/reports) are lists of potential problems
+in MusicBrainz. These reports are generated daily by
```
Comment (mwiencek) [inline HACKING.md]: Nit: by the

File: HACKING.md [hunk 2]
```diff
@@ -171,6 +171,26 @@ Now you should be able to use it to run the tests:
...
+*[daily.sh](https://github.com/metabrainz/musicbrainz-server/blob/master/admin/cron/daily.sh)*
+script.
+
+You can generate all reports using *[RunReports.sh](https://github.com/metabrainz/musicbrainz-server/blob/master/admin/RunReports.pl)*
```
Comment (mwiencek) [inline HACKING.md]: Nit: using the

**Comment by mwiencek** [comment]
:shipit:

**Comment by gentlecat** [comment]
:+1:

**Pull request merged**